app/forms/idv/api_image_upload_form.rb

```diff
@@ -98,10 +98,12 @@ def post_images_to_client
     end
 
     def front_image_bytes
+      return unless front
       @front_image_bytes ||= front.read
```

Comment:
would this be a good time to use nil-safe operator?
```suggestion
      @front_image_bytes ||= front&.read
```

Reply:
ah, but then `.try(:read)` would work

Reply:
I refactored slightly to add a `readable?` function to check that we can read/hash the contents.